.github/workflows/atomate-terraform.yml

```diff
@@ -0,0 +1,48 @@
+name: Workflow Ducks Web Page
+
+on:
+  push:
+    branches:
+      - dev
+
+env:
+  ENVIRONMENT: "develop"
+  STATE_KEY: "Tfstate-ducks-server"
+
+jobs:
+  set-up-terraform-backend:
+    runs-on: ubuntu-latest
+        
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+              
+      - name: Configure AWS Credentials
+        id: config_aws_creds
+        uses: aws-actions/configure-aws-credentials@v2

```

Comment:
a new version of this action has been released, please check, i think that is v3 the last one

Reply:
Yes, there is a new one that has been pending to be explained for a long time and it was with the use of roles.
I have this link: https://aws.amazon.com/es/blogs/security/use-iam-roles-to-connect-github-actions-to-actions-in-aws/
but can you explain to me please

Reply:
ok

Reply:
You approved it but this part is missing.